package/contents/ui/main.qml

```diff
@@ -268,30 +300,30 @@ Item {
 			if (isOnDesktop) {
 				return Math.ceil(output.contentWidth)
 			} else if (isHorizontal && plasmoid.configuration.useFixedWidth) {
-				return plasmoid.configuration.fixedWidth * units.devicePixelRatio
+				return plasmoid.configuration.fixedWidth * Kirigami.Units.devicePixelRatio
 			} else { // isHorizontal || isVertical
 				return Math.ceil(output.implicitWidth)
 			}
 		}
 		Layout.minimumWidth: isHorizontal ? itemWidth : -1
 		Layout.fillWidth: isVertical
-		Layout.preferredWidth: itemWidth // Panel widget default
-		// width: itemWidth // Desktop widget default
+		Layout.preferredWidth: itemWidth // Panel root default
+		// width: itemWidth // Desktop root default
 		// onItemWidthChanged: console.log('itemWidth', itemWidth, 'implicitWidth', output.implicitWidth, 'contentWidth', output.contentWidth)
 
 		readonly property int itemHeight: {
 			if (isOnDesktop) {
 				return Math.ceil(output.contentHeight)
 			} else if (isVertical && plasmoid.configuration.useFixedHeight) {
-				return plasmoid.configuration.fixedHeight * units.devicePixelRatio
+				return plasmoid.configuration.fixedHeight * Kirigami.Units.devicePixelRatio
```

Comment:
I need to check if `Kirigami.Units.devicePixelRatio` and `Screen.devicePixelRatio` have a difference.